### ISSUE TITLE
App crashes if you press the ESC key

### ISSUE THREAD

**Issue by mself** [quicky2000/tartini#56]
On Mac, the app crashes if you press the ESC key.

I see this code in `MainWindow::keyPressEvent()`:
```
#ifdef MYDEBUG
        case Qt::Key_Escape:
            g_data->stopAndWaitAudioThread();
            qApp->closeAllWindows();
        break;
#endif // MYDEBUG
```
But the app crashes with `EXC_BAD_ACCESS` in the call to `makeCurrent()` in `VibratoTunerWidget::~VibratoTunerWidget()`:
```
VibratoTunerWidget::~VibratoTunerWidget()
{
  // Remove display lists
  makeCurrent();

  glDeleteLists(m_dial, 1);
  glDeleteLists(m_needle, 1);
}
```

I don't understand what the value is of this "debug" feature?

**Comment by quicky2000** [comment]
I wonder it could be to have way to quit when GUI is frozen (like for me in play and record due to RtAudio callback looping)

**Comment by quicky2000** [comment]
App return properly on my side

**Comment by quicky2000** [comment]
Closed by #60
